BootloaderCommonPkg/Library/ContainerLib/ContainerLib.c

```diff
@@ -230,6 +230,31 @@ LocateComponentEntryFromContainer (
   return CompEntry;
 }
 
+/**
+  This function returns the hash alg type from auth type.
+
+  @param[in] AuthType    Authorization Type.
+
+  @retval         Hash Algorithm Type.
+
+**/
+HASH_ALG_TYPE GetHashAlg(

```

Comment:
Coding style requires different line between HASH_ALG_TYPE and GetHashAlg.

Reply:
Can you fix this  before we merge ?

Reply:
@Maurice, Sachin and myslef think adding a parameter for pubkeyhash type  may not be required. DoHashVerify need to be separate methods. One method type can query hashalg type  from hash store for pubkey and stage components.

Let us discuss tomorrow and close it.

Reply:
Fixed the coding style issue.